extensions/spark/kyuubi-spark-lineage/src/main/scala/org/apache/kyuubi/plugin/lineage/helper/SparkSQLLineageParseHelper.scala

```diff
@@ -94,6 +111,8 @@ trait LineageParser {
         k -> AttributeSet(attrs.collect {
           case attr if relationAttrSet.contains(attr) =>
             attr.withName(Seq(tableName, attr.name).filter(_.nonEmpty).mkString("."))
```

Comment:
can we change the type of `tableName` to `Seq[String]`, and use `attr.withQualifier(tableName)`. At caller side, we can do `attr.qualifiedName`